measure/src/main/kotlin/com/alliander/open/measure/Measure.kt

```diff
@@ -117,8 +117,16 @@ data class Measure<U : Units>(val amount: BigDecimal, val units: U) : Comparable
         val absoluteFactor = (factor `in` base).abs()
         return dividend.roundToMultiple(absoluteFactor, roundingMode) * base
     }
-
 }
 
 private fun BigDecimal.roundToMultiple(factor: BigDecimal, roundingMode: RoundingMode): BigDecimal =
-        this.divide(factor, 0, roundingMode) * factor
+    this.divide(factor, 0, roundingMode) * factor
+
+operator fun <U : Units> BigDecimal.times(m: Measure<U>): Measure<U> =
```

Comment:
In some cases you want to multiply your measure with some dimensionless value. e.g. when calculating the effectiveness of an consumer equipment on the congested element `effectiveness * (4 * megaWatt)`)

Reply:
Wasnt this functionality already present in measure 1.3? From MeasureKt.class
```
public fun <A : com.alliander.measure.Units, B : A> baseUnits(first: A, second: B): A { /* compiled code */ }

public operator fun <A : com.alliander.measure.Units, B : A> A.compareTo(other: B): kotlin.Int { /* compiled code */ }

public operator fun <U : com.alliander.measure.Units> java.math.BigDecimal.times(units: U): com.alliander.measure.Measure<U> { /* compiled code */ }

public operator fun <U : com.alliander.measure.Units> kotlin.Double.times(units: U): com.alliander.measure.Measure<U> { /* compiled code */ }

public operator fun <U : com.alliander.measure.Units> kotlin.Float.times(units: U): com.alliander.measure.Measure<U> { /* compiled code */ }

public operator fun <U : com.alliander.measure.Units> kotlin.Int.times(units: U): com.alliander.measure.Measure<U> { /* compiled code */ }

public operator fun <U : com.alliander.measure.Units> kotlin.Long.times(units: U): com.alliander.measure.Measure<U> { /* compiled code */ }


```

Reply:
No, what you are referring to is a multiplication with a Unit --> `13 * kiloWatt` 

What this does is a multiplication with a Meaure:
```
val measure: Measure<Power> = 4 * megaWatt
val newMeasure: Measure<Power> = 3 * measure // 12 MW 
```